src/ucs/profile/nvtx/nvtx.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) NVIDIA Corporation. 2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#ifdef HAVE_CUDA
```

Comment:
do we need different macro for nvtx header? since it needs specific cuda version